src/constants.ts

```diff
@@ -0,0 +1,3711 @@
+/* eslint-disable tsdoc/syntax */
+/* eslint-disable @typescript-eslint/no-empty-interface */
+
+import type { Snowflake } from 'discord-api-types/globals';
+import type {
+	APIMessage,
+	APIPartialChannel,
+	APIPartialGuild,
+	APIUser,
+	APIVoiceState,
+	ChannelType,
+	GatewayActivity,
+	OAuth2Scopes,
+} from 'discord-api-types/v10';
+
+export const RPCVersion = '1';
```

Comment:
Should be screaming snake case:

```suggestion
export const RPC_VERSION = '1' as const;
```